io.github.trevorsandy.LPub3D.json

```diff
@@ -0,0 +1,274 @@
+        "if test -d ${FLATPAK_DEST}/lpub3d_linux_3rdparty/ldview-4.4; then echo \"LDView staged to ${FLATPAK_DEST}/lpub3d_linux_3rdparty/ldview-4.4.\"; fi"
+      ]
+    },
+    {
+      "name": "povray",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cd unix && chmod +x prebuild3rdparty.sh && ./prebuild3rdparty.sh",
+        "./configure COMPILED_BY=\"Trevor SANDY <trevor.sandy@gmail.com> using FlatHub for LPub3D.\" --prefix=${LP3D_DIST_DIR_PATH} LPUB3D_3RD_PARTY=yes --with-libsdl2 --with-boost-libdir=${FLATPAK_DEST}/lib --enable-watch-cursor",
+        "make",
+        "make check",
+        "make install"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/povray.git",
+          "commit": "54b65529090b4161193df24f0d846046b1495f0b"
+        }
+      ],
+      "modules": [
+        {
+          "name": "boost",
+          "buildsystem": "simple",
+          "cleanup": [
+            "/include",
+            "/lib/libboost_*.a"
+          ],
+          "build-commands": [
+            "./bootstrap.sh --prefix=${FLATPAK_DEST} --with-libraries=date_time,thread",
+            "./b2 install variant=release link=shared runtime-link=shared --prefix=/app -j $FLATPAK_BUILDER_N_JOBS"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.bz2",
+              "sha256": "686affff989ac2488f79a97b9479efb9f2abae035b5ed4d8226de6857933fd3b"
+            }
+          ]
+        },
+        {
+          "name": "openexr",
+          "modules": [
+            {
+              "name": "ilmbase",
+              "config-opts": [ "--disable-static" ],
+              "sources": [
+                {
+                  "type": "archive",
+                  "url": "https://download.savannah.nongnu.org/releases/openexr/ilmbase-2.2.1.tar.gz",
+                  "sha256": "cac206e63be68136ef556c2b555df659f45098c159ce24804e9d5e9e0286609e"
+                },
+                {
+                  "type": "shell",
+                  "commands": [ "cp -p /usr/share/automake-*/config.{sub,guess} ." ]
+                }
+              ]
+            }
+          ],
+          "config-opts": [ "--disable-static", "--disable-ilmbasetest" ],
+          "cleanup": [ "/bin", "/share/doc", "/lib/libIlmImfUtil*" ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://download.savannah.nongnu.org/releases/openexr/openexr-2.2.1.tar.gz",
+              "sha256": "8f9a5af6131583404261931d9a5c83de0a425cb4b8b25ddab2b169fbf113aecd"
+            },
+            {
+              "type": "shell",
+              "commands": [ "cp -p /usr/share/automake-*/config.{sub,guess} ." ]
+            }
+          ]
+        },
+        {
+          "name": "libtiff",
```

Comment:
libtiff 4.1 is in the runtime.

I removed it in my test build and it works.

Is this really necessary?

Reply:
> Is this really necessary?

No. I did not see any indication that libtiff was in the runtime, so I added it. Like you said, “figuring out what needs to be build is sometime a guess job”.

Anyway, I will be happy to remove it as it requires a good bit of cleanup.

Cheers,